docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
```

Comment:
To get your **API key** you will need to create an account on the [Tequila](tequila.kiwi.com) website. The registration is free and gives you also access to other features of Tequila platform.

Reply:
done